tests/karate/src/test/java/be/election/electionOpen.feature

```diff
@@ -0,0 +1,122 @@
+
+CODE CLIMATE OPTIONS
+Show issues
+Show test coverage
+Highlight covered lines
```

Comment:
What is this ?

Reply:
I do not have a slightest idea, I removed it now